package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "prop-types": "^15.8.1",
```

Comment:
in pure js , the linter is expecting you to use this "prop - validation - library"

This validation is demanded by the linter only for js projects becasue  Typescript is already validating the properties from the `props`...

2 options: 

1. supress the linter warning (NOT recommended)
2. install (`npm i prop-types`) and call the library as following